spdx/writers/rdf.py

```diff
@@ -1016,10 +1016,11 @@ def write(self):
         for package_node in self.packages():
             package_triple = (doc_node, self.spdx_namespace.describesPackage, package_node)
             self.graph.add(package_triple)
-        """# Add relationship
-        relate_node = self.relationships()
-        relate_triple = (doc_node, self.spdx_namespace.relationship, relate_node)
-        self.graph.add(relate_triple)"""
+        # Add relationship
+        for relate_node in self.relationships():
+            relate_triple = (doc_node, self.spdx_namespace.relationship, relate_node)
+            self.graph.add(relate_triple)
```

Comment:
I tried this out using
```
$ convertor -f tag data/SPDXTagExample -o output.rdf
```
and the output contained this, which doesn't seem to make sense (relationships should be separate entities):
```
    <ns1:relationship>
      <ns1:Relationship rdf:about="SPDXRef-DOCUMENT">
        <ns1:relationshipType>CONTAINS</ns1:relationshipType>
        <ns1:relationshipType>DESCRIBES</ns1:relationshipType>
        <ns1:relationshipType>COPY_OF</ns1:relationshipType>
        <ns1:relatedSpdxElement>SPDXRef-File</ns1:relatedSpdxElement>
        <ns1:relatedSpdxElement>DocumentRef-spdx-tool-1.2:SPDXRef-ToolsElement</ns1:relatedSpdxElement>
        <ns1:relatedSpdxElement>SPDXRef-Package</ns1:relatedSpdxElement>
      </ns1:Relationship>
    </ns1:relationship>
```

I think this either needs more work, or should be handled as a separate issue - whichever way you prefer!